netket/models/deepset.py

```diff
@@ -12,6 +12,7 @@
 
 from netket.utils import deprecate_dtype
 from netket.hilbert import ContinuousHilbert
+import netket as nk
```

Comment:
maybe best to import only MLP from nk.models ?

Reply:
Nice catch. Yeah tou should not import netket inside netket because it might wreak havok in the future